toolchain/sem_ir/formatter.h

```diff
@@ -27,6 +36,7 @@ class Formatter {
   const File& sem_ir_;
   // Caches naming between Print calls.
   InstNamer inst_namer_;
+  ShouldFormatEntityCallback should_format_entity_;
```

Comment:
Maybe either whitespace or its own comment to make clearer that this isn't part of the caching comment above? e.g.:

```suggestion

  ShouldFormatEntityCallback should_format_entity_;
```

```suggestion
  // See `ShouldFormatEntityCallback`.
  ShouldFormatEntityCallback should_format_entity_;
```

Reply:
Reordered before `inst_namer_`; I think that's also better, to keep the values specified in the constructor call separate from the computed members.